client/src/components/layout/DestinationShow.js

```diff
@@ -26,16 +30,33 @@ const DestinationShow = (props) => {
     getDestination();
   }, []);
 
+  const reviewsList = reviews.map(reviewItem => {
+    return ( 
+        <ReviewTile 
+            key={reviewItem.id} 
+            title={reviewItem.title}
+            content={reviewItem.content}
+            rating={reviewItem.rating}
+        />
+    )
+  })
+
   return (
-    <div className="destination">
-      <h1>{name}</h1>
-      <p>{description}</p>
-      <ul>
-        <li>{type}</li>
-        <li>{location}</li>
-        <li>{website}</li>
-      </ul>
-    </div>
+    <>  
+      <div className="destination">
+        <h1>{name}</h1>
+        <p>{description}</p>
+        <ul>
+          <li>{type}</li>
+          <li>{location}</li>
+          <li>{website}</li>
+        </ul>
+        <Link to={`/destinations/${destination.id}/new-review`}>Submit a New Review</Link>
```

Comment:
OPTIONAL - it is an authenticated route, however can consider conditionally rendering this link only when a user is logged in, when they're not logged it it might display a message "you must sign in to leave a review"